app/src/debug/java/io/github/droidkaigi/confsched2018/di/NetworkModule.kt

```diff
@@ -53,6 +55,20 @@ import javax.inject.Singleton
                 .build()
     }
 
+    @Singleton @Provides @Named("sponsor") @JvmStatic
```

Comment:
👍

Reply:
How about using Qualifier?
https://github.com/DroidKaigi/conference-app-2018/blob/master/app/src/main/java/io/github/droidkaigi/confsched2018/di/CustomQualifier.kt

Reply:
Thanks for reviews! That's true!
 But jsons have been deployed officially, so this code disappeared. 😂

Reply:
Fixed. https://github.com/DroidKaigi/conference-app-2018/pull/176/commits/5b748249b0fa03a18fdfa680cf3ab541d64d8766